src/main/java/com/icthh/xm/ms/dashboard/web/rest/DefaultProfileResource.java

```diff
@@ -75,6 +77,7 @@ public DefaultProfileResource(DefaultProfileService defaultProfileService,
     @PutMapping("/default-profiles")
     @Timed
     @PreAuthorize("hasPermission({'id': #defaultProfile.id, 'newDefaultProfile': #defaultProfile}, 'defaultProfile', 'DEFAULT_PROFILE.UPDATE')")
+    @PrivilegeDescription("Privilege to updates an existing defaultProfile")
```

Comment:
replace defaultProfiles to more informative message. Describe what is defaultProfiles.